src/test/java/SetTest.java

```diff
@@ -0,0 +1,57 @@
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.HashSet;
+import java.util.Set;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+@DisplayName("Set Collection에 대한 학습 테스트")
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Nested
+    @DisplayName("요구사항 1")
+    class Size_테스트 {
+        @Test
+        @DisplayName("Set의 size() 메소드를 활용해 Set의 크기를 확인하는 학습테스트를 구현한다.")
+        void Size_테스트() {
+            assertThat(numbers.size()).isEqualTo(3);
+        }
+    }
+
+    @Nested
+    @DisplayName("요구사항 2")
+    class Contains_테스트1 {
+        @ParameterizedTest
+        @ValueSource(ints = {1, 2, 3})
+        @DisplayName("ParameterizedTest를 활용해 Set에 1, 2, 3의 값이 존재하는지 확인한다.")
```

Comment:
테스트케이스가 변경될 경우 DisplayName이 변경되지 않도록 변경해보면 좋겠어요!
만약 테스트데이터의 정보를 확인하고 싶다면

```suggestion
        @ParameterizedTest(name = "ParameterizedTest를 활용해 Set에 {0}의 값이 존재하는지 확인한다.")
        @ValueSource(ints = {1, 2, 3})
        @DisplayName("ParameterizedTest를 활용해 Set에 1, 2, 3의 값이 존재하는지 확인한다.")
```

와 같이 ParameterizedTest의 기능을 활용할 수 있습니다 :)

Reply:
앞으로 `DisplayName`을 작성할 때, 테스트데이터에 따라 변경되지 않을 수 있게 만들어 보겠습니다! 제안해 주신 방법도 입력값에 따라 성공여부를 판단할 때, 한눈에 보여서 좋을것 같습니다.